src/main/java/core/basesyntax/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.DataConverter;
+import core.basesyntax.strategy.FruitTransaction;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    public List<FruitTransaction> convertToTransaction(List<String> rawData) {
+        List<FruitTransaction> fruitTransactions = new ArrayList<>();
+        for (int i = 1; i < rawData.size(); i++) {
+            String[] parts = rawData.get(i).split(",");
+            FruitTransaction.Operation operation = FruitTransaction.Operation.fromCode(parts[0]);
+            String fruit = parts[1];
+            int quantity = Integer.parseInt(parts[2]);
+            fruitTransactions.add(new FruitTransaction(operation, fruit, quantity));
+        }
```

Comment:
The current implementation assumes that the input data is always well-formed. Consider adding validation to handle potential parsing errors, such as checking if `parts` has the expected length and catching `NumberFormatException` when parsing the quantity.